### ISSUE TITLE
Wrong design

### ISSUE THREAD

**Issue by caicher** [robEllenberg/gcode-syntax#2]
When i set the design **Deep Black** it will be horrible

**Comment by Heratiki** [comment]
Yeah I had the same issues so I had to go in and change everything so that it had a back blackground under the different Styler setups.
